### ISSUE TITLE
mas outdated shows apps that were just updated

### ISSUE THREAD

**Issue by agersoncgps** [mas-cli/mas#92]
I just ran
```
mas upgrade 409201541
Upgrading 1 outdated application:
Pages (6.1.1)
==> Downloading Pages
==> Installed Pages
```

Pages shows as updated
![screenshot 2017-05-25 16 12 03](https://cloud.githubusercontent.com/assets/336441/26468560/ef2043d2-4164-11e7-8d32-cd206f40069c.png)

But it still shows as needing an update:
```
mas outdated
715768417 Microsoft Remote Desktop (8.0.27253 -> 8.0.39)
425264550 Blackmagic Disk Speed Test (3.0 -> 3.1)
512464723 Alinof Timer (4.1.0 -> 4.1.2)
409183694 Keynote (7.1 -> 7.1.1)
409203825 Numbers (4.1 -> 4.1.1)
497799835 Xcode (8.3 -> 8.3.2)
408981434 iMovie (10.1.4 -> 10.1.5)
409201541 Pages (5.6.2 -> 6.1.1)
```

**Comment by nick-f** [comment]
Possibly related to https://github.com/mas-cli/mas/issues/76? Have you tried launching the Mac App Store and then re-running `mas`?

**Comment by phatblat** [comment]
Should be fixed now in [1.8.1](https://github.com/mas-cli/mas/releases/tag/v1.8.1). @agersoncgps can you confirm?

**Comment by chris-araman** [comment]
Additional discrepancies in `outdated` resolved in #346.
